prime-router/docs/design/design/yaml-validation/yaml-value-validation.md

```diff
@@ -0,0 +1,20 @@
+# YAML Value Validation
+
+## Approach
+
+To validate the structure of our YAML files, we will use the Konform validation library. We will define data classes
+corresponding to our yaml files and be able to define simple validations on them.
```

Comment:
validations is a bad word here I think, it's too vague

Reply:
I'll be more specific.